src/cascadia/WindowsTerminal/AppHost.cpp

```diff
@@ -1351,7 +1353,7 @@ void AppHost::_handleMoveContent(const winrt::Windows::Foundation::IInspectable&
         windowBoundsReference = inDips.to_winrt_rect();
     }
 
-    _windowManager.RequestMoveContent(args.Window(), args.Content(), args.TabIndex(), windowBoundsReference);
+    _windowManager.RequestMoveContent(targetWindow, args.Content(), args.TabIndex(), windowBoundsReference);

```

Comment:
This change seems unrelated? The previous code had the benefit that the 3 `args` arguments map more clearly to the function call.

Reply:
op yea that was a side effect of the first attempt at a fix